src/Integration/Suppression/SonarQubeIssuesProvider.cs

```diff
@@ -109,6 +118,7 @@ public IEnumerable<SonarQubeIssue> GetSuppressedIssues(string projectGuid, strin
             this.initialFetch?.Wait(MillisecondsToWaitForInitialFetch);
 
             if (this.suppressedFileIssues == null ||
+                this.suppressedFileIssues.Count == 0 ||
```

Comment:
Miscellaneous: added a check for empty lists here as a short-circuit.
Now that the tests use an empty list rather than `null`, they hit this new check.

Reply:
This changes the logic a bit. 
Previously if the suppressedFileIssues was empty, we would get to the second block of modules, and try to return module issues. We won't do that anymore.
Is there a reason for this change? How long has it been since modules were deprecated? I think the code in the current state might be slightly confusing, perhaps it would be better to do a pre-commit that removes the modules.

Reply:
Good point. I'll revert the change.
I don't know whether the module stuff is still relevant or not, but it's unrelated to the branch awareness sprint. I'm not going to touch it now.